src/client/components/Table/Table.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled'
+import { ReactNode } from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { getMargins, Margins } from '../utils/margins'
+
+type WithChildrenProps = {
+  children?: ReactNode
+}
+
+export type TableProps = {
+  minWidth?: string
+  maxWidth?: string
+  fullWidth?: boolean
+} & Margins
+
+const Table = styled.table(
+  ({ minWidth, maxWidth, fullWidth, ...props }: TableProps) => ({
+    borderCollapse: 'collapse',
+    borderSpacing: 0,
+    minWidth,
+    maxWidth,
+    width: fullWidth ? '100%' : 'auto',
+    ...getMargins(props),
+  }),
+)
+
+const TableHead = styled.thead()
+
+const TableRow = styled.tr()
+
+const TableBody = styled.tbody()
+
+const getTableCellStyles = ({ align = 'left', color }: TableCellProps) => ({
+  textAlign: align,
+  minWidth: '4rem',
+  color,
+})
+
+type TableCellProps = WithChildrenProps & {
+  align?: 'center' | 'left' | 'right'
+  color?: keyof typeof colorsV3 | undefined | string
+}
+
+const TableCell = styled.td((props: TableCellProps) => ({
+  ...getTableCellStyles(props),
+  padding: '0.5rem 0',
+
+  borderBottom: `1px solid ${colorsV3.gray300}`,
+}))
+
+const TableHeaderCell = styled.th((props: Omit<TableCellProps, 'color'>) => ({
```

Comment:
Just for consistency maybe we can call this `TableHeadCell`?

Or go all the way and render based on context like MUI: https://mui.com/material-ui/api/table-cell/ ;)

> The component renders a <th> element when the parent context is a header or otherwise a <td> element.

Reply:
Yeah, I was considering the conditional rendering based on parent.. I can try it with a simple `createContext` perhaps